package.json

```diff
@@ -1,61 +1,20 @@
 {
-    "name": "@datadog/ui-extensions-sdk",
-    "version": "0.23.0",
-    "description": "Datadog UI Extensions - Official JavaScript SDK ",
+    "name": "@datadog/apps",
+    "description": "Datadog Apps Monorepo",
```

Comment:
These are basically unused, since it's a workspace. We can remove them or do something else if we'd like.